.github/workflows/bb_containers.yml

```diff
@@ -161,7 +166,7 @@ jobs:
         run: |
           cd ${{ env.WORKDIR }}
           cat ${{ matrix.dockerfile }} common.Dockerfile >/home/runner/work/Dockerfile
-          cp -r qpress /home/runner/work
```

Comment:
Do not remove qpress since it's used by some mtr tests.

Reply:
It's preserved.